### DISCUSSION TITLE
Override LogOut page

### DISCUSSION THREAD

**Discussion by dor1202** [OrchardCMS/OrchardCore#12818] [Q&A] [answered]
Hi.
I'm trying to override the view for `Logout.cshtml` page,
I saw issue #11644 but with no success.
I placed the `Logout.cshtml` page in : `customTheme->Views->OrchardCore.OpenId->Access` folder

**Comment by ManOguaR** [comment]
ANYONE HAS MANAGE TO OVERRIDE OPENID LOGOUT?

#11644 offers NO SOLUTION

and the talk derives into useless information... SPA or NOT, login and logout views are served by orchard... 

![image](https://github.com/user-attachments/assets/b3b85e25-cd47-4ff5-982b-ba150e5f8746)

Can anyone tell the community WHERE TO PUT OPENID VIEW OVERRIDES IN THEME?

**Reply by ManOguaR** [reply, marked as answer]
FINALLY GOT SOLUTION:

I was using a custom theme selector, and i nedd only themes to be only ENABLED for many overrides.

BUT... to override OpenId views, Theme MUST BE SET TO SITE THEME... and CAN ONLY use Site Theme for OpenId View Overrides :(

al least with my current theme selector settings :P

**Reply by Piedone** [reply]
Thank you for coming back with a solution!
